### DISCUSSION TITLE
middleware/kafka/user-guide/view/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1896] [Announcements]
# middleware/kafka/user-guide/view/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/middleware/kafka/user-guide/view/

**Comment by liyuerich** [comment]
这个图有点旧了，现在有日志查看，kafka配置参数，zookeeper配置参数

**Comment by ModetaNiu** [comment]
Fixed in PR #4781
